README.md

```diff
@@ -2,30 +2,26 @@
 [![codecov](https://codecov.io/gh/ansible-community/tox-ansible/branch/master/graph/badge.svg)](https://codecov.io/gh/greg-hellings/tox-ansible)
 [![PyPI version](https://badge.fury.io/py/tox-ansible.svg)](https://badge.fury.io/py/tox-ansible)
 
-tox-ansible-collection
-======================
+tox-ansible
+===========
 
-This plugin for tox auto-generates tox environments for running Ansible
+This plugin for tox auto-generates tox environments for running Molecule scenarios. Optionally, you can
```

Comment:
I think that "Optionally, you can" doesn't belong here. It doesn't look like it reads properly with that.